lib/matplotlib/__init__.py

```diff
@@ -160,6 +160,7 @@
 # definitions, so it is safe to import from it here.
 from . import _api, _version, cbook, _docstring, rcsetup
 from matplotlib._api import MatplotlibDeprecationWarning
+from matplotlib.colors import _color_sequences as color_sequences
```

Comment:
I assume this was at the bottom just to group it with the colormap imports, but now I'm using it within `rcsetup` it needs to be imported early.

Reply:
Before I moved it I got an attribute error that `mpl` did not have `color_sequences`.  I guess I could also have imported `_color_sequences` directly from `colors` in `rcsetup`?

Reply:
I see, these circular imports are awkward. For now, it's ok to move this up.

Side-note: We fundamentally have an issue with the validators: `mpl.rcParams` should be available early because some modules explicitly depend on it during startup. It's populated from `matplotlibrc` using the validators in `rcsetup`, but the validators may need information from other modules. So there's a delicate dependency on who goes first in the import order.